commands/behold.js

```diff
@@ -0,0 +1,25 @@
+const Augur = require("augurbot"),
```

Comment:
This really belongs in the `meme.js` module. Having a new command module for each command would generate an overwhelming number of modules. Let's group them by function. It can be added to the `meme.js` module by chaining this `.addCommand()` to the end of the existing `.addCommand()` in that file.